### PR TITLE
Update fips test to ovn

### PR DESCRIPTION
**Pull request by svetsa-rh**
Change from SDN to OVN

Addresses:
https://issues.redhat.com/browse/PERFSCALE-2790
https://issues.redhat.com/browse/PERFSCALE-2792

### CODE REVIEW

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 1]
```diff
@@ -3864,18 +3864,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.15"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-24nodes-sdn-fips
-  reporter_config:
```
Comment (paigerube14) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3868]: looks like the reporter config stuff is being removed here, lets re-add

**Comment by paigerube14** [comment]
Things look good here, would probably want to validate the installer workflow installs properly before merging

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml (hunk 1 above)
Comment (svetsa-rh) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3868]: done

**Comment by svetsa-rh** [comment]
@paigerube14 Any idea how to resolve
Error from server (Forbidden): secrets "pull-secret" is forbidden: User "system:serviceaccount:ci-op-vnynn77w:node-density-3nodes-ovn-fips" cannot get resource "secrets" in API group "" in the namespace "openshift-config"

**Comment by paigerube14** [comment]
> @paigerube14 Any idea how to resolve Error from server (Forbidden): secrets "pull-secret" is forbidden: User "system:serviceaccount:ci-op-vnynn77w:node-density-3nodes-ovn-fips" cannot get resource "secrets" in API group "" in the namespace "openshift-config"
> 
> and there is a core dump too

Hm no, never seen that before

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-3nodes-ovn-fips pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-node-density-3nodes-ovn-fips

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-3nodes-ovn-fips pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-node-density-3nodes-ovn-fips

**Comment by svetsa-rh** [comment]
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/47838/rehearse-47838-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-3nodes-ovn-fips/1748105326298664960 - failed in cerberus check for golang pods, similar to https://redhat-internal.slack.com/archives/CH76YSYSC/p1705591945628259

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/47838/rehearse-47838-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-node-density-3nodes-ovn-fips/1748105326369968128 - failed due to prow resource constraints

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-node-density-3nodes-ovn-fips

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-3nodes-ovn-fips
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-node-density-3nodes-ovn-fips

rerun tests after removing fips-check-fips-or-die step

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-node-density-3nodes-ovn-fips
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-node-density-3nodes-ovn-fips

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-node-density-3nodes-ovn-fips

**Comment by paigerube14** [comment]
Need to update workflow as well: https://github.com/openshift/release/blob/f244d4bc6864c3c582cc316b94cac3eb7a5d4569/core-services/ci-chat-bot/workflows-config.yaml#L386

**Comment by svetsa-rh** [comment]
@liangxia @AlexNPavel @bradmwilliams @hoxhaeris Can I get an approval here?

**Pull request merged**